doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-rest-api-endpoints-ref.adoc

```diff
@@ -29,27 +29,5 @@ For the full list of {KIE_SERVER} REST API endpoints and descriptions, use one o
 NOTE: By default, the Swagger web interface for {KIE_SERVER} is enabled by the `org.kie.swagger.server.ext.disabled=false` system property. To disable the Swagger web interface in {KIE_SERVER}, set this system property to `true`.
 
 ifdef::PAM,JBPM[]
-[discrete]
-== Endpoint requirements
-
-Note the following requirements for some of the {KIE_SERVER} REST API endpoints:
-
-* *Process images:* For API access to process images, the system property `<storesvgonsave enabled="true"/>` must be configured for your {PRODUCT} project in `$SERVER_HOME/standalone/deployments/{URL_COMPONENT_CENTRAL}.war/org.kie.workbench.KIEWebapp/profiles/jbpm.xml`. This property is set to `true` by default. If the API is not working with process images, set it to `true` in the file, restart your {KIE_SERVER}, modify the relevant process and save it, and then build and deploy your project. This property enables SVG images to be stored so that they can be retrieved by the {KIE_SERVER} REST API.
-
-* *Custom queries:* Some custom query requests with the {KIE_SERVER} REST API require a query `mapper` definition to map the query results to concrete objects. You can implement your own query result mappers or use the mappers provided with {PRODUCT}. The query mappers in {PRODUCT} are similar to other object-relational mapping (ORM) providers, such as Hibernate, which maps tables to entities. For example, you can use the `org.jbpm.kie.services.impl.query.mapper.ProcessInstanceQueryMapper`, also registered as `ProcessInstances`, in custom query endpoints for returning process instance data.
-+
---
-Example POST endpoint with `ProcessInstances` mapper parameter:
-----
-http://localhost:8080/kie-server/services/rest/server/queries/definitions/jbpmProcessInstances?mapper=ProcessInstances
-----
```

Comment:
This example was omitted, is it no longer needed?

Reply:
The example, and the entire file in fact, was omitted because of the general policy not to include specific endpoints int he document

Reply:
@mramendi Understand. So how does a user get to know about the endpoint format? Is it specified generally somewhere else?

Reply:
The endpoints are available in the Swagger UI. 

There is a clear reason for this documentation decision. Endpoints can change, and Swagger changes with them. So, the document explains concepts, then the user looks at Swagger to see the details.

Is this ok to merge now?